.github/workflows/ci.yml

```
@@ -14,3 +14,19 @@ jobs:
         run: cargo clippy --workspace --all-targets -- -D warnings
       - name: Cargo test
         run: cargo test --workspace
+  generate-rust:
+    name: Generate Rust crate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Clean
+        run: rm src/bindings.rs
+      - name: Generate bindings
+        run: cd bindings_generator && cargo run
+      - name: Upload crate source
+        uses: actions/upload-artifact@v4
+        with:
+          name: crate-source
+          path: src/
+      - name: Diff generated Rust code
+        run: test -z "$(git status --porcelain)" || (echo "::error::Generated files are different, please regenerate with cargo run -p generate_bindings!"; git status; false)
```

```suggestion
        run: test -z "$(git status --porcelain)" || (echo "::error::Generated files are different, please regenerate with cargo run -p generate_bindings!"; git diff; false)

```

(And the trailing newline to make github happy)

---

Fixed in latest commit